src/io/perun/core.clj

```diff
@@ -11,15 +12,15 @@
   "Return metadata on files. Files metadata is a list.
    Internally it's stored as a map indexed by `:path`"
   [fileset]
-  (-> fileset meta +meta-key+ vals))
+  (keep +meta-key+ (vals (:tree fileset))))
```

Comment:
`(keep :key coll)` is like `(filter identity #(map :key coll))` https://clojuredocs.org/clojure.core/keep